solanum-tuberosums/bangazonOrientationAPI
# Added AdminCustomerSerializer so authenticated users see extra field.

## Associated Ticket
[Restrict Non-Super Users](https://github.com/solanum-tuberosums/bangazonOrientationAPI/issues/35)

## Description 
Created new serializer and overwrote a method in view_customer.py

**TO TEST:**
* ```python manage.py builddb```
* ```python manage.py createsuperuser```
* ```python manage.py runserver```
* BEFORE logging in, make sure you CANNOT see the 'date_created' field under Customer table
* AFTER logging in, make sure you CAN see the 'date_created' field under Customer table

## Migrations

<p style="text-align: center;"><center>🥔</center></p>
